crits/samples/templates/samples_detail.html

```diff
@@ -75,6 +76,16 @@ <h3 class='titleheader'>
                 </td>
             </tr>
             <tr>
+                <td class="key">Filepaths</td>
```

Comment:
My intention was to have a single filepath entry associated with a submitted sample, but populating the filepaths list.